samples/msal-browser-samples/ExpressSample/.env.e2e

```diff
@@ -0,0 +1,5 @@
+CLIENT_ID=b5c2e510-4a17-4feb-b219-e55aa5b74144
```

Comment:
Non-blocking but this can also be configured ADO-side if you want to avoid checking in the env file.